api/cmd/portainer/main.go

```diff
@@ -99,11 +99,11 @@ func initJWTService(dataStore portainer.DataStore) (portainer.JWTService, error)
 		return nil, err
 	}
 
-	if settings.UserSessionTimeout == "" {
-		settings.UserSessionTimeout = portainer.DefaultUserSessionTimeout
-		dataStore.Settings().UpdateSettings(settings)
+	userSessionTimeout := settings.UserSessionTimeout
```

Comment:
this is a possible change in functionality, should be reviewed. Maybe it needs to go in a different PR